cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRNonSignedoutMutationDataReader.java

```diff
@@ -114,20 +114,20 @@ public void open(ExecutionContext ec) throws ItemStreamException {
         }
         // add header and filename to write to for writer
         ec.put("mutationHeader", new ArrayList(header));
-        ec.put("mafFilename", CVRUtilities.UNFILTERED_MUTATION_FILE);
+        ec.put("mafFilename", CVRUtilities.NONSIGNEDOUT_MUTATION_FILE);
         summaryStatistics.printSummaryStatistics();
     }
 
     private void loadMutationRecordsFromJson(CVRData cvrData) {
         List<MutationRecord> recordsToAnnotate = new ArrayList<>();
         for (CVRMergedResult result : cvrData.getResults()) {
             String sampleId = result.getMetaData().getDmpSampleId();
-            int sampleSNPCount = result.getAllCvrSnps().size();
+            int countNonSignedoutSampleSnps = result.getAllNonSignedoutCvrSnps().size();
             String somaticStatus = result.getMetaData().getSomaticStatus() != null ? result.getMetaData().getSomaticStatus() : "N/A";
-            for (CVRSnp snp : result.getAllCvrSnps()) {
+            for (CVRSnp snp : result.getAllNonSignedoutCvrSnps()) {
```

Comment:
here is a nice speedup -- we annotated mutations just once ... the signed out mutations get annotated in their file, and the non-signedout mutations get annotated in their file [no overlap]